src/Resources/assets/styles/_tailwind.scss

```diff
@@ -39,6 +39,14 @@
     .whatwedo-utility-link {
         @extend .whatwedo-utility-heading-4;
         @apply text-primary-500;
+        text-decoration: underline solid transparent;
+        text-underline-offset: 4px;
```

Comment:
Should `text-underline-offset` that be relative (em or %) to text size rather than an absolute pixel value?